project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
```

Comment:
Could we elaborate a bit more here? I think either it is our policy to always use app.zenhub.com, or it's a suggestion, and we should make it clear which with the wording.

Personally i think it's just a suggestion, as in "ZenHub features can be accessed on github.com with the browser plugin, or at app.zenhub.com."

Personally I find it nicer to use app.zenhub.com for *boards* specifically, but every other page (issues, pull requests etc.) Is nicer within github itself. But this duality is too confusing to be an official recommendation.

Reply:
Yeah I'll reword and remove personality 1000111

Reply:
It's not the personality per se that I disapprove of. Just please mould your personality to fit in with The Official Version Of Events.

Reply:
Affirmative

Reply:
tidied up a bit